src/applications/edu-benefits/feedback-tool/config/form.js

```diff
@@ -35,10 +35,10 @@ const {
   serviceDateRange,
 } = fullSchema.properties;
 
-const { assistance, programs, school } = educationDetails;
+const { assistance, programs, school } = educationDetails.properties;
 const { address: schoolAddress, name: schoolName } = school.properties;
-const domesticSchoolAddress = schoolAddress.oneOf[0];
-const internationalSchoolAddress = schoolAddress.oneOf[1];
+const domesticSchoolAddress = schoolAddress.anyOf[0];
```

Comment:
Very well done 👏 , great catch using anyOf instead of oneOf.

Reply:
This caused so much confusion this morning. Then things still didn't work when I switch to `anyOf` because of a stupid copy/paste error I'd introduced in my code :(